.github/workflows/build-and-test.yml

```diff
@@ -291,14 +291,20 @@ jobs:
       - name: Run Unit Tests
         if: startsWith( matrix.go-version, '1.22' ) != true
         run: make gotest GROUP=${{ matrix.group }}
-      - name: Run Unit Tests With Coverage
-        if: startsWith( matrix.go-version, '1.22' ) # only run coverage on one version
-        run: make gotest-with-cover GROUP=${{ matrix.group }}
+      - name: Run Unit Tests With JUnit and Coverage
+        if: startsWith( matrix.go-version, '1.22' ) # only run junit/coverage on one version
+        run: make gotest-with-junit-and-cover GROUP=${{ matrix.group }}
       - uses: actions/upload-artifact@v4
         if: startsWith( matrix.go-version, '1.22' ) # only upload artifact for one version
         with:
           name: coverage-artifacts-${{ matrix.go-version }}-${{ matrix.runner }}-${{ matrix.group }}
           path: ${{ matrix.group }}-coverage.txt
+      - uses: actions/upload-artifact@v4
+        if: startsWith( matrix.go-version, '1.22' ) # only upload artifact for one version
```

Comment:
```suggestion
        if: startsWith( matrix.go-version, '1.23' ) # only upload artifact for one version
```

Reply:
I was looking at this again, I'm bit confused actually.

It's using 1.22 in all other places in this file... if we need to update to 1.23, would it make sense to do all of them at once? Do we need to do it here?

Reply:
Should I resolve the comment then? Or commit the suggestion?

Reply:
I think we can mark this as resolved yes :)